src/commands/project/add-service.ts

```diff
@@ -0,0 +1,103 @@
+import commander from "commander";
+import fs from "fs";
+import yaml from "js-yaml";
+import path from "path";
+import shell from "shelljs";
+import mkdirp from "mkdirp";
```

Comment:
should keep on using shelljs instead. you can do `shell.mkdir('-p', 'your-dir')`